formats/shared/definitions/_specialist_document.jsonnet

```diff
@@ -412,6 +415,287 @@
       },
     },
   },
+  uk_market_conformity_assessment_body_metadata: {
+    type: "object",
+    additionalProperties: false,
+    properties: {
+      uk_market_conformity_assessment_body_name: {
+        type: "string",
+      },
+      updated_at: {
+        type: "string",
+      },
+      uk_market_conformity_assessment_body_number: {
+        type: "string",
+      },
+      uk_market_conformity_assessment_body_type: {
+        type: "array",
+        items: {
+          type: "string",
+          enum: [
+            "approved-body",
+            "recognised-third-party-organisation",
+            "technical-assessment-body",
+            "user-inspectorate",
+          ],
+        },
+      },
+      uk_market_conformity_assessment_body_registered_office_location: {
+        type: "string",
+        enum: [
+          "united-kingdom",
```

Comment:
The order of these seems confusing. Worth clarifying whether this is desired?

Reply:
updated